apps/fs_perf/example.yaml

```diff
@@ -0,0 +1,25 @@
+- name: example
+  fs: [SFS, EXT4, XFS]
+  device-path: /dev/sdX
+  runs-nb: 100
+  smp: [5, 1, 12]
+  files-info:
+    small-files-nb: [1, 2, 4, 8, 16, 1024]
+    big-files-nb: [1, 2, 4, 8, 16]
+  probabilities:
+    small-prob: [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]
+    write-prob: [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]
+  operations-info:
+    limits:
+      written-data-limit: 400M
+      read-data-limit: 400M
+      op-nb-limit: 10000
+    parallelism: [64, 128, 256]
+    seq-writes: [false, true]
+    small-op-size-range: [[0M, 1023k], [1023k, 1023k]]
```

Comment:
How does it work? You end-up with Cartesian product of all combination provided here and such number of tests are run or what?

Reply:
Yes